src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -31,24 +27,48 @@ public override void Accept(OperationVisitor visitor)
             return visitor.VisitAddressOfExpression(this, argument);
         }
     }
+    /// <summary>
+    /// Represents an expression that creates a pointer value by taking the address of a reference.
+    /// </summary>
+    internal sealed partial class AddressOfExpression : AddressOfExpressionBase, IAddressOfExpression
```

Comment:
I think we use the pattern BaseXXX, not XXX base.  i.e. BaseTypeDeclarationSyntax